docs/basics.rst

```diff
@@ -404,3 +405,11 @@ production environments for reproducible builds.
     If you'd like a ``requirements.txt`` output of the lockfile, run ``$ pipenv lock -r``.
     This will include all hashes, however (which is great!). To get a ``requirements.txt``
     without hashes, use ``$ pipenv run pip freeze``.
+
+
+☤ Few notable differences
+-------------------------
+
+- ``pip install pipenv`` is installing pipenv as a tool.
```

Comment:
Why mention `pip install pipenv` here, `pip` is a different tool from `pipenv`.

Reply:
The statements are useful as the users are confused between these options and maintainers feel documentation should define the items properly. Makes sense?

Reply:
Clarifying the differences between these commands is valuable. We already have some notes in https://docs.pipenv.org/en/latest/advanced/#using-pipenv-for-deployments but it is way too simple. You should add more doc there about the differences between `pipenv sync`, `pipenv install --deploy`, `pipenv install --ignore-pipfile` and other similar commands, and what is recommended under particular situation.

Reply:
I would advise you remove this section here.

Reply:
I reffing the comment https://github.com/pypa/pipenv/pull/3939#discussion_r322034530 in the ticket I created.